src/spreadPath.js

```diff
@@ -0,0 +1,29 @@
+import { curry, pathOr, merge, dissoc } from 'ramda';
+
+/**
+ * Merges object under path with root object, removing path to object.
```

Comment:
`Spreads object under property path onto provided object.`

IMHO we don't  need to tell that it removed the object from the path. Spread is by definition a new composition of the object.